### ISSUE TITLE
Loop through sqlTable scripts and execute them

### ISSUE THREAD

**Issue by dolson334** [dolson334/bantor#18]
-1.  create a batch file-
2. in the batch file create a loop that will execute every file in you bantor/database/{newDirYouCreated}
3.  This will create the schema for every table we need.

**Comment by dolson334** [comment]
FYI, you will also need to create a new database using java then.. Shouldnt be too hard. 

You should start a pull request so I can comment on and help.

**Comment by dolson55** [comment]
Having a little trouble getting started here.  What's the best plan of attack? Idk if working in eclipse is going to work.

**Comment by dolson334** [comment]
Ill shoot something your way sometime today

**Comment by dolson334** [comment]
1. I added a new project called dbBuilder (C:\bantor\database\build\dbBuilder) with a main class called dbBuilder.java.  
2. I created a batch file called buildDb.bat (C:\bantor\database\build\buildDb.bat)
3. Running bat file executes the dbBuilder.jar
3. Every time you make a change to dbBuilder.java you need to export to a new jar
4.  To export, in eclipse File --> Export --> JAR File --> NEXT --> Select dbBuilder --> export the destination jar FIle to bantor\database\build\dbBuilder.jar --> NEXT --> Select buildDb as the main class.  

Super easy

**Comment by dolson55** [comment]
Probably a dumb question but I'm looking at the psedocode (where you have the instruction) in buildDB.java, and step 1 one says build the database.... I've already done that correct??

**Comment by dolson334** [comment]
Not a dumb question.  

Nope you have not built it yet.

**Comment by dolson334** [comment]
Actually you do have a database that you backed up from on your server your right!  But what we are trying to do hear is build another database every time that batch file is run... So just keep creating new databases well clean them out later.

**Comment by dolson334** [comment]
https://www.mkyong.com/jdbc/how-to-run-a-mysql-script-using-java/ 

Gives a great example of how to read in and execute a .sql file

**Comment by dolson55** [comment]
Okay so pulled dbBuilder from git... there were a couple of bugs that I needed to fix (mostly build path issues). Should I run SqlConnBuilder last?

**Comment by dolson334** [comment]
when you execute the batch file does it successfully create the database named DAVE?

**Comment by dolson55** [comment]
Okay sorry for the delay on this had kind of a crazy weekend.  I just made a bit of a breakthrough on this issue so you can expect me to have this done or at least very close to done by tonight if not early tomorrow.

**Comment by dolson334** [comment]
when you run the batch file are you able to create the database?

**Comment by dolson55** [comment]
no I am not.  I'm getting a message that says Error: Unable to access jarfile dbBuilder.jar

**Comment by dolson55** [comment]
Also what is the purpose of the dbBuilder.jar file?

**Comment by dolson334** [comment]
the jar file is the java code all bundles up.  Essentially, the batch file just executes the jar

**Comment by dolson55** [comment]
Okay that makes sense now. So that error i mentioned above is preventing me from creating the database.

**Comment by dolson55** [comment]
Do I have to create a new connection to connect to the DB?

**Comment by dolson334** [comment]
No you shouldnt.  I think I got the batch file running now (pull what I just did).  When you export use runnable jar instead of jar. It will create a database called "DAVE".  ONce you run this once the next time it will fail since dave already exists. So delete the dave database each time.  

So pull and make sure when you run that batch file you get the newly created DB when you run the batch.  We can skype tomorrow as well .
